assets/css/_ladder.scss

```diff
@@ -2,6 +2,11 @@
   grid-row-start: m-ladder;
   margin: 0 auto;
   position: relative;
+
+  .__react_component_tooltip {
```

Comment:
The `__` makes me think this might be private, like in python conventions. Are there docs saying this is the right way to select it?

Also, I'm surprised that these properties aren't the default, given that it looks a lot like the demo at https://wwayne.github.io/react-tooltip/ . Is this selector even needed?

Reply:
This is the best solution that I've seen. And yes, the style is very similar to the default, that's why only these very small tweaks are necessary.

Reply:
The default is so close, is it even worth styling it to change the border radius from 3px to 4px? Can we just use the default?

If we do want this css, let's put it somewhere more general, like app, base, or ui_kit. I assume we'll want all our tooltips to look the same.

Reply:
Moved it to app.